BlueDBOnDisk/src/main/java/org/bluedb/disk/segment/path/SegmentPathTimeUnits.java

```diff
@@ -0,0 +1,24 @@
+package org.bluedb.disk.segment.path;
+
+public class SegmentPathTimeUnits {
+	public static long ONE_MILLI = 1L;
+	public static long SIX_SECONDS = 6_000L;
+	public static long ONE_MINUTE = 60_000L;
+	
+	public static long ONE_HOUR = 3_600_000L;
+	public static long TWO_HOURS = ONE_HOUR * 2;
+	public static long FOUR_HOURS = ONE_HOUR * 4;
+	public static long SIX_HOURS = ONE_HOUR * 6;
+	public static long EIGHT_HOURS = ONE_HOUR * 8;

```

Comment:
I originally had that as a supported time segment size but removed it so that all the segment sizes could be multiples of each other. Do you want me to delete the unused ones?

Reply:
I'd prefer the unused ones be deleted, but I don't feel that strongly about it.

Reply:
I deleted the unused ones